subsys/bluetooth/Kconfig.scan

```diff
@@ -116,6 +116,45 @@ config BT_SCAN_MANUFACTURER_DATA_CNT
 
 endif
 
+config BT_SCAN_CONN_ATTEMPTS_FILTER
+	bool "Connection attempts filter"
+	help
+	 Connection attempts filter. This filter limits the number of attempts
+	 to connect to the device.
+
+if BT_SCAN_CONN_ATTEMPTS_FILTER
+
+config BT_SCAN_CONN_ATTEMPTS_FILTER_LEN
+	int "Connection attempts filtered device count"
+	default 2
+	help
+	  The maximum number of the filtered devices by
+	  the connection attempts filter.
+
+config BT_SCAN_CONN_ATTEMPTS_CNT
+	int "Connection attempts count"
+	default 2
+	help
+	 Connection attempts count. Defines how many times the device will
```

Comment:
actually: `how many times the device will try to connect with a given peripheral.`

Reply:
Changed